.github/workflows/pr-build.yml

```diff
@@ -21,6 +21,10 @@ jobs:
           - 12.x
           - 14.x
           - 15.x
+        include:
+          - os: ubuntu-latest
+            node-version: 14.x
```

Comment:
Why node 14.x and not 15.x?

Reply:
Odd numbered major versions of node are effectively permanent RCs for the next "real" even-numbered major version release. The odd-numbered versions are only supported for 6 months, and generally not used in production. You can read more here: https://nodejs.org/en/about/releases/